kubernetes/compose/v1beta2/doc.go

```diff
@@ -0,0 +1,6 @@
+// Api versions allow the api contract for a resource to be changed while keeping
+// backward compatibility by support multiple concurrent versions
+// of the same resource
+
+// Package v1beta2 is the second version of the stack, containing a structured spec
```

Comment:
Same question here for positioning of the Package description (should likely be the first line in this file

Reply:
Done